lib/driverProviders/mock.ts

```diff
@@ -4,10 +4,15 @@
  * server.
  */
 import * as q from 'q';
+import {DeferredExecutor, Executor, Session, WebDriver} from 'selenium-webdriver';
```

Comment:
Removing `DeferredExecutor` and `Executor`. The Session constructor is used at `getNewDriver`.